get-history.js

```diff
@@ -54,21 +50,50 @@ function exportActivity() {
       // component; grab its current parents in case we need them as a fallback
       const id = act.targets[0].driveItem.name;
       const shortId = id.split("/")[1];
-      const parentIter = DriveApp.getFileById(shortId).getParents();
-      let parents = [];
-      while (parentIter.hasNext()) {
-        parents.push("items/" + parentIter.next().getId());
+
+      try
+      {
+        const parentIter = DriveApp.getFileById(shortId).getParents();
+        let parents = [];
+        while (parentIter.hasNext()) {
+          parents.push("items/" + parentIter.next().getId());
+        }
+        act.targets[0]._d2g_parents = parents;
+        seenFiles[id] = true;
+      }
+      catch(e)
+      {
+        console.log(e);
+        console.log("Could not parse file by Id!");
       }
-      act.targets[0]._d2g_parents = parents;
-      seenFiles[id] = true;
+      
     }
 
-    act.actors[0].user._d2g_info = users[userId];
-
     if (lastProgressReport + 1000 * 10 <= Date.now()) {
```

Comment:
Does this make it so you can pick up where you started off? Or run on a cron/CI job I'm guessing?

Reply:
This makes it so that we don't throw away file data just because the user data is unusable